types/tizen-common-web/index.d.ts

```diff
@@ -1,84 +1,14699 @@
+    /**
+     * Retrieves information about the member files in _ArchiveFile_.
+     *
+     * The errorCallback is launched with these error types:
+     *
+     * *   UnknownError: In case of any error
+     *
+     *
+     * @privilegeLevel public
+     * @privilegeName http://tizen.org/privilege/filesystem.read
+     *
+     * @param onsuccess Callback method to be invoked when information about all the files in the archive is successfully retrieved.
+     * @param onerror Callback method to be invoked when an error occurs.
+     *
+     * @returns Task ID which can be used to cancel the operation with abort().
+     *
+     * @throw WebAPIException with error type TypeMismatchError, if parameter is of the wrong type.
+     * @throw WebAPIException with error type InvalidStateError, if ArchiveFile is not open.
+     * @throw WebAPIException with error type InvalidAccessError, if the file mode is "w" or "a".
+     * @throw WebAPIException with error type SecurityError, if the application does not have the privilege to call this method or the application does not have privilege to access the storage. For more information, see [Storage privileges](#StorageRemark).
+     */
+    getEntries(onsuccess: ArchiveFileEntryArraySuccessCallback, onerror?: ErrorCallback | null): number;
+    /**
+     * Retrieves information about _ArchiveFileEntry_ with the specified name in _ArchiveFile_.
+     *
+     * The errorCallback is launched with these error types:
+     *
+     * *   NotFoundError: If _ArchiveFileEntry_ with the specific name does not exist
+     * *   UnknownError: In case of any other error
+     *
+     *
+     * @privilegeLevel public
+     * @privilegeName http://tizen.org/privilege/filesystem.read
+     *
+     * @param name Name of _ArchiveFileEntry_ to extract.
+     * @param onsuccess Callback method to be invoked when a file matched with the given name is found.
+     * @param onerror Callback method to be invoked when an error occurs.
+     *
+     * @returns Task ID which can be used to cancel the operation with abort().
+     *
+     * @throw WebAPIException with error type TypeMismatchError, if parameter is of the wrong type.
+     * @throw WebAPIException with error type InvalidStateError, if ArchiveFile is not opened.
+     * @throw WebAPIException with error type InvalidAccessError, if the file mode is "w" or "a".
+     * @throw WebAPIException with error type SecurityError, if the application does not have the privilege to call this method or the application does not have privilege to access the storage. For more information, see [Storage privileges](#StorageRemark).
+     */
+    getEntryByName(name: string, onsuccess: ArchiveFileEntrySuccessCallback, onerror?: ErrorCallback | null): number;
+}
+export interface ArchiveFile {
```

Comment:
Another case of incorrectly declaring the *static* (i.e., constructor) side of the class on an interface that represents the *instance*. Please remove.